warehouse/query-core/src/main/java/datawave/query/jexl/visitors/ExpandMultiNormalizedTerms.java

```diff
@@ -198,16 +195,16 @@ private Object expandRangeForNormalizers(LiteralRange<?> range, JexlNode node) {
             
             // Avoid extra parens around the expansion
             if (1 == aliasedBounds.size()) {
-                return aliasedBounds.get(0);
+                // we assume the Optional container is not empty as we have already created a new BoundedRange from the node
+                return Arrays.stream(aliasedBounds.toArray()).findFirst().get();
             } else {
-                List<ASTReferenceExpression> var = JexlASTHelper.wrapInParens(aliasedBounds);
+                List<ASTReferenceExpression> var = JexlASTHelper.wrapInParens(aliasedBounds.stream().collect(Collectors.toList()));
```

Comment:
Probably more efficient: new ArrayList(aliasedBounds);

Reply:
Updated in last commit